src/AutoMapper/QueryableExtensions/Impl/SourceInjectedQuery.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace AutoMapper.QueryableExtensions.Impl
+﻿using System.Security.Cryptography.X509Certificates;

```

Comment:
I believe that's for WCF Encryption Certificate.  Not sure why that needs to be in solution though.